uninstall.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Plugin reset and uninstall cleanup.
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit;
+
+// Bail if not uninstalling or reseting the plugin.
+if ( ! defined( WP_UNINSTALL_PLUGIN ) && empty( $googlesitekit_reset ) ) {
+	return;
+}
+
+global $wpdb;
+
+$key = '%googlesitekit%';
+
+// Delete options and transients.
+$wpdb->query(
+	$wpdb->prepare( "DELETE FROM $wpdb->usermeta WHERE meta_key LIKE %s OR meta_key LIKE %s", $key, 'sitekit_authentication' )
```

Comment:
I think we should rely on prefixes specifically like outlined in the IB - `googlesitekit_%`, `{$wpdb->get_blog_prefix()}googlesitekit_%`, `_transient_googlesitekit_%` etc.

Specifically, what is this `sitekit_authentication`?

Reply:
Agreed 👍